src/ocean_emulators/constants.py

```diff
@@ -122,11 +166,17 @@ def construct_metadata(data: xr.Dataset) -> Dict[str, Dict[str, str]]:
                 "units": data[var].units,
             }
         except AttributeError:
-            logging.info(f"{var} has no long_name or units attribute")
-            metadata[str(var)] = {
-                "long_name": "Unknown",
-                "units": "Unknown",
-            }
+            if var in DEFAULT_METADATA.keys():
+                metadata[str(var)] = DEFAULT_METADATA[str(var)]
+            elif str(var).split("_")[0] in DEFAULT_METADATA.keys():
+                metadata[str(var)] = DEFAULT_METADATA[str(var).split("_")[0]]
```

Comment:
Consider using the walrus operator here:
```suggestion
            elif (key := str(var).split("_")[0]) in DEFAULT_METADATA.keys():
                metadata[str(var)] = DEFAULT_METADATA[key]
```